<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorEnableWidget</class>
 <widget class="QWidget" name="SensorEnableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>592</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>592</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>气象仪</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>就绪</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>电罗经</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEnableGyro">
   <property name="geometry">
    <rect>
     <x>287</x>
     <y>150</y>
     <width>54</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
  <widget class="QWidget" name="wReadyGypo" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>154</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
background-image: url(:/images/推进器启用设置-日.png);
border-width:0;
border-style:outset;</string>
   </property>
  </widget>
  <widget class="QWidget" name="wReadyWind" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>214</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
background-image: url(:/images/推进器启用设置-日.png);
border-width:0;
border-style:outset;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEnableWind">
   <property name="geometry">
    <rect>
     <x>287</x>
     <y>210</y>
     <width>54</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border-width:0;
border-style:outset;</string>
   </property>
   <property name="text">
    <string>启用</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
